magit-stgit.el

```diff
@@ -607,18 +604,13 @@ the To, Cc, and Bcc fields for all patches."
 
 ;;; Mode
 
-(defvar magit-stgit-mode-map
-  (let ((map (make-sparse-keymap)))
-    (define-key map "/" 'magit-stgit-popup)
-    map))
-
-(magit-define-popup-action 'magit-dispatch-popup ?/ "StGit" 'magit-stgit-popup)
+(transient-append-suffix 'magit-dispatch "@"
+  '("/" "StGit" magit-stgit))
```

Comment:
I had some trouble with this `transient-append-suffix` call. I think the "@" location only exists if the `magit-annex` extension is loaded, so it seems like we need to append relative to a location defined by magit itself.

One possibility to consider would be after "Z" (Worktree) which immediately preceeds "!" (Run).

Also, take a look at what `magit-annex` does:

```elisp
;;;###autoload
(eval-after-load 'magit
  '(progn
     (define-key magit-mode-map "@" 'magit-annex-dispatch)
     (transient-append-suffix 'magit-dispatch '(0 -1 -1)
       '("@" "Annex" magit-annex-dispatch))))
```

So they get their "@" into the `magit-dispatch` transient with a location of `'(0 -1 -1)` which, I believe, means the first group's last subgroup's last item.

Reply:
That's interesting, because I'm not using the `magit-annex` package and don't have it loaded either. For me what appears under `@` is "Forge", but I haven't been able to pinpoint who's setting this yet.

Using the the coordinate approach and the particular coordinates you've described makes sense to me though, as it seems like that particular place within Magit's transient is where at least other 2 "non-alphanumeric key bindings that invoke commands to interface with external tools" are.

Reply:
Ok, I've found the culprit -- it was `evil-collection` that was setting it to `@`. Thanks for the pointer to how `magit-annex` sets up the key binding though.